<?xml version="1.0"?>

<launch>
   <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor.gazebo.xacro" />

   <group ns="uav1">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav1" />
       <arg name="tf_prefix" value="uav1" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="1.0" />
       <arg name="y" value="1.5" />
       <arg name="z" value="0.5" />
     </include>
     <node name="pose_estimation" pkg="hector_pose_estimation" type="pose_estimation" required="true" />
   </group>

   <group ns="uav2">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav2" />
       <arg name="tf_prefix" value="uav2" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="-1.0" />
       <arg name="y" value="-1.5" />
       <arg name="z" value="0.5" />
     </include>
     <node name="pose_estimation" pkg="hector_pose_estimation" type="pose_estimation" required="true" />
   </group>

   <group ns="uav3">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav3" />
       <arg name="tf_prefix" value="uav3" />
       <arg name="model" value="$(arg model)" />
       <arg name="x" value="0.0" />
       <arg name="y" value="-15.0" />
       <arg name="z" value="1.0" />
     </include>
     <node name="pose_estimation" pkg="hector_pose_estimation" type="pose_estimation" required="true" />
   </group>

</launch>
